docs/extensions/usage/README.md

```diff
@@ -1,3 +1,14 @@
 # Using Extensions
 
-TBD
+The start using Lens Extensions go to `Lens > Extensions` in the application menu. This brings you to the screen that helps you manage your extensions. There are three ways to install extensions. Either you can download a `.tar` file of the extension and find it with the file picker or, if it is hosted on the web, you can give Lens the URL and it can download it for you. The third way is to move the extension into your `~/.k8slens/extensions` folder and Lens will automatically detect it.
```

Comment:
In addition to `~/.k8slens/extensions`, I think `C:\Users\<user>\.k8slens\extensions` (for windows) need to be mentioned here?